public class Exception1 {
    public static void main(String[] args) {
        
        //Try

        int[] m={1,22,3,4};
        try {
            System.out.println(m[8]);
        } catch (Exception e) {
            
        }
        System.out.println("Pradip");
    }
}
